firmware/arduino/chessboard/chessboard.ino

```diff
@@ -193,25 +202,37 @@ void loop()
     if (validateMessageFromPi(receivedMessagePtr))
     { 
         // Sends acknowledgement
-        sendMessageToPi(currentState, moveCount, errorCode);
+        sentMessage[0] = currentState;
+        sentMessage[1] = errorCode;
+        sentMessage[2] = moveCount;
+        sendMessageToPi(sentMessage);
 
         makeMove(receivedMessagePtr);
     }
     // Sends move success/error
```

Comment:
It would be a good idea to edit the comment to clarify that you're re-setting the variables and sending them out since they have the potential to change from makeMove. Otherwise it just looks like repeated code.